core/utils/poly-look/src/visualisations/charts/flow-charts/sankeyDiagram/sankeyDiagram.js

```diff
@@ -89,12 +97,13 @@ export class SankeyDiagram extends Chart {
   }
 
   _createSVG() {
-    return d3
+    const svg = d3
```

Comment:
It is repeated once as I overwrite createSVG here as it doesnt work otherwise. The gradient step makes most sense at the place where the svg is created i think

Reply:
But I was thinking more along the lines of 

```javascript
const svg = new polySvg(this._selector)
```

And that would include, at least, the first three lines below, possibly also the ones that establish height and basic styles.

Reply:
(Mind you, this is a general thought, not within the scope of this PR)

Reply:
Yeah it's a good one as well! Didn't ever see defs not being gradients until now, but for sure this will pop up